lib/generators/active_fedora/config/fedora/templates/.fcrepo_wrapper

```diff
@@ -2,3 +2,4 @@
 port: 8984
 enable_jms: false
 fcrepo_home_dir: tmp/fcrepo4-development-data
+download_dir: /tmp
```

Comment:
That'd certainly be better (and cross platform..)... although isn't this the same as the default behavior upstream?